src/Types/PlutusAddress.purs

```diff
@@ -0,0 +1,192 @@
+module Types.PlutusAddress where
+
+import Prelude
+import Data.Maybe (Maybe(..))
```

Comment:
We've been using explicit imports for all constructors (even common types like `Maybe`)